WooCommerce/Classes/Yosemite/DefaultStoresManager.swift

```diff
@@ -4,6 +4,8 @@ import Yosemite
 import enum Networking.DotcomError
 import class Networking.UserAgent
 import class Networking.WordPressOrgNetwork
+import KeychainAccess
+import WidgetKit
```

Comment:
Since we use it only on a single line, can we narrow import to `import class WidgetKit.WidgetCenter`?